app/web/features/auth/locales/en.json

```diff
@@ -223,6 +225,7 @@
       "error_message": "Failed to recover account: {{message}}",
       "button_text": "Recover your account",
       "success": "Your account has been recovered, you can log back in. We're excited to have you back!"
-    }
+    },
+    "missing_token": "Error: missing token"
   }
 }
```

Comment:
Actually, could you look through the diff in this file and try to remove extra changes please?